Client/Frontend/Reader/ReaderViewLoading.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+ - License, v. 2.0. If a copy of the MPL was not distributed with this
+ - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<html>
+
+    <body>
+        <h1 id="message">TODO Loading Interstitial</h1>
+    </body>
+
+    <script>
+        var numberOfChecks = 10;
+
+        function triggerCheck() {
+            if (numberOfChecks--) {
+                setTimeout(function() { checkIfContentIsAvailable(); }, 1000);
+            } else {
+                var e = document.getElementById("message")
+                if (e != null) {
+                    e.innerText = "There was a problem loading the content. Click here to load the original page.";
+                }
+            }
+        }
+
+        function checkIfContentIsAvailable() {
```

Comment:
This all feels really hacky for what we're trying to do. Can we not set up a callback for the `ReadabilityOperation`, then have Swift load the reader page instead of HTML polling? It'd be cleaner IMO, but also faster (since any time between the load and the next timeout is currently wasted).